resources/views/playarea.blade.php

```diff
@@ -3,15 +3,19 @@
     <h3>Playground</h3>
     <p><strong>level:</strong> {{ Auth::User()->current_level }}</p>
 
-    @if ($question->type == 'photo')
-        <img src="{{ $question->source_url }}" alt="">
-    @elseif($question->type == 'video')
+    @if ($question->attachment->type == 'image')
+        <img src="/question_attachments/{{ $question->attachment->id }}">
+    @elseif($question->attachment->type == 'video')
         <div>
-            video
+            <video width="480" height="200" controls>
+                <source src = "/question_attachments/{{ $question->attachment->id }}" type="video/mp4">
+            </video>
         </div>
     @else
         <div>
-            audio
+            <audio controls>
+                <source src = "/question_attachments/{{ $question->attachment->id }}" type="audio/mpeg">
+            </audio>
         </div>
```

Comment:
wrap this in `elseif($question->attachment->type == 'audio')` and use "else" condition to display some error message